libs/components/packages/src/schematics/rules/apply-skyux-stylesheets-to-workspace.ts

```diff
@@ -20,7 +20,7 @@ export function applySkyuxStylesheetsToWorkspace(): Rule {
           const target = project.targets.get(targetName);
 
           /*istanbul ignore else*/
-          if (target) {
+          if (target && target.options) {
```

Comment:
```
if (target?.options) {
```

Reply:
I was having trouble with getting coverage to cover the bill operator there as a branch. Splitting it up got coverage to pass. It was minor so I just did it this way.